### ISSUE TITLE
Chromedriver Error

### ISSUE THREAD

**Issue by tatwell** [formulafolios/ann-arbor#1]
After following steps in README, when I run script I get error:

> selenium.common.exceptions.WebDriverException: Message: Service chromedriver unexpectedly exited. Status code was: 127

To reproduce:

```
$ python app.py audit --crawl httpbin.org

[several lines of output skipped]

selenium.common.exceptions.WebDriverException: Message: Service chromedriver unexpectedly exited. Status code was: 127
```

Note: I did not have this problem on my Mac laptop where I installed chromedriver using brew. It was a problem on my Ubuntu system.

**Comment by tatwell** [comment]
### Workaround

I was able to workaround this issue like so:

To fix, I needed to explicitly set `executable_path` as suggested [here](https://stackoverflow.com/a/58816881). I changed [this line](https://github.com/formulafolios/ann-arbor-internal-beta/blob/9eec91c666ab0989e4522a9a892dd89294575d41/models/axe_audit.py#L300) to:

    driver = webdriver.Chrome(chrome_options=chrome_options, executable_path='/usr/local/bin/chromedriver')

Script succeeded:

```
$ python app.py audit --crawl httpbin.org
...
aXe Site Audit Summary
----------------------
domain:         httpbin.org
pages:          1
violations:     5
\- errors:      5
\- warnings:    0

Top Templates by Violations:
forms: 5

Top Subtemplates by Violations:
forms/post: 5

created:        2020-04-17 00:06:33
runtime:        0:00:03.606384

Violations CSV: projects/ann-arbor/audits/httpbin-org/httpbin-org-site-all-violations.csv

```

I am not going to commit this. I am going to explore a more resilient fix: perhaps making the path setting an optional config setting.

**Comment by tatwell** [comment]
### Path Issue
I discovered a conflict with my Chromedriver path caused by rbenv on my system.

```
$ chromedriver --version
rbenv: chromedriver: command not found

The `chromedriver' command exists in these Ruby versions:
  2.5.1

$ which chromedriver
/home/klenwell/.rbenv/shims/chromedriver
```

I followed steps in this Stack Overflow answer to remove the chromedriver shim (hope I didn't need it):

- https://stackoverflow.com/a/52411524/9381758

Now when I check chromedriver version:

```
$ which chromedriver
/usr/local/bin/chromedriver

$ chromedriver --version
ChromeDriver 81.0.4044.69 (6813546031a4bc83f717a2ef7cd4ac6ec1199132-refs/branch-heads/4044@{#776})
```

However, when I run the script, I get a new error:

```
$ python app.py audit --crawl httpbin.org

[several lines of output skipped]

selenium.common.exceptions.SessionNotCreatedException: Message: session not created: This version of ChromeDriver only supports Chrome version 81
```

I'll open a new issue for this. :weary:

**Comment by tatwell** [comment]
### Resolved
Resolved with Issue #2 by adding `webdriver-manager` Python package.
